NuGet.config

```diff
@@ -4,14 +4,19 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-aspnetcore-tooling -->
+    <add key="darc-int-dotnet-aspnetcore-tooling-aec2954" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-aspnetcore-tooling-aec2954c/nuget/v3/index.json" />
```

Comment:
These changes should be reverted right?

Reply:
Nope, they can stay. Lower in the file there are lines that disable these feeds in public builds